test/integration/krane_deploy_test.rb

```diff
@@ -1712,6 +1712,18 @@ def test_deploy_resource_using_generate_name_succeeds
     ])
   end
 
+  def test_deploy_fails_fast_when_erb_is_present
```

Comment:
How is this different from https://github.com/Shopify/kubernetes-deploy/pull/635/files#diff-ab79bd0231cddd4a971c2848fad61e1cL1647 ?

Reply:
hmmm, `test-wise` they seems to test the same thing, if I'm not misreading it I feel we should delete this test.

Reply:
Yeah oops! I figured I'd add a test for my change, didn't check if there was one there already. My bad!